### PR TITLE
Update page header and favicon

### PR DESCRIPTION
**Pull request by josh13w**
Tests run: `npm run test`
```
Test Suites: 8 passed, 8 total
Tests:       11 passed, 11 total
Snapshots:   0 total
Time:        2.249 s, estimated 3 s
Ran all test suites.
```

Manually tested by confirming that header is using the new image and favicon uses correct image.

### CODE REVIEW

File: sample_tracking/app/login/page.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default function LogInSignUpPage() {
 
   useEffect(() => {
     hideNavBar();
-    hideTopBar();
+    // hideTopBar();
```
Comment (benwulfe) [inline sample_tracking/app/login/page.tsx:45]: should either delete or uncomment

Comment (benwulfe) [reply sample_tracking/app/login/page.tsx:45]: or leave with a comment indicating why its left there

File: sample_tracking/app/top_bar.tsx [hunk 2]
```diff
@@ -35,14 +37,14 @@ export default function TopBar() {
...
-                    router.push('/login');
-                } else {
+                if (user) {
                     setUserData({
                         name: user.displayName!,
                         photoUrl: user.photoURL!,
                         email: user.email!,
                     } as UserData)
+                } else {
+                    setUserData(null);
```
Comment (benwulfe) [inline sample_tracking/app/top_bar.tsx:47]: i would have thought /login would still be valid but include the header? no?

File: sample_tracking/app/top_bar.tsx [hunk 3]
```diff
@@ -59,9 +61,9 @@ export default function TopBar() {
             }
         });
 
-    })
+    }, [])
 
-    if (!userData) return;
+    // if (!userData) return;
```
Comment (benwulfe) [inline sample_tracking/app/top_bar.tsx:66]: remove or explain...

**Review by benwulfe — CHANGES_REQUESTED** [review]
this seems like a slightly more dangerous set of changes to me.
Can you share the link and we can give it more testing?

**Review by benwulfe — CHANGES_REQUESTED** [review]
please address comments

File: sample_tracking/app/top_bar.tsx (hunk 2 above)
Comment (josh13w) [reply sample_tracking/app/top_bar.tsx:47]: Yes this is just saying that there is no logged in user so we won't show the user profile image/letter in the top right.

**Review by benwulfe — CHANGES_REQUESTED** [review]
can we add the link to docs here or is that a big change?

File: sample_tracking/app/top_bar.tsx (hunk 2 above)
Comment (josh13w) [reply sample_tracking/app/top_bar.tsx:47]: We can test with the link created in the PR. But I've tested with both seeing the top bar when logged in and logged off and the profile image is shown only when logged in.

**Review by benwulfe — APPROVED** [review]

**Pull request merged**